### ISSUE TITLE
Matrix row/column addition

### ISSUE THREAD

**Issue by mateogianolio** [mateogianolio/vectorious#64]
Proposal:

> Adds a multiple of one row to another inside a matrix.

``` javascript
// (Matrix, Number, Number, Number) => (Matrix)
Matrix.rowAdd = function (matrix, i, j, scalar)

// (this, Number, Number, Number) => (this)
Matrix.prototype.rowAdd = function (i, j, scalar)
```

Example:

``` javascript
var m = new Matrix([[1, 2], [3, 4]]);
m.rowAdd(0, 1, 10);
// [[31, 42], [3, 4]]
```

_Same for columns._

**Comment by christianbender** [comment]
@mateogianolio I will work on this. For example a method for the class ```Matrix``` 
```js
rowAdd(dest : number, source : number, scalar : number) : Vector
```
* dest (destination) is the row for adding the other row that was multiplied with the scalar.
* source : is the row that would be multiplied with the scalar (for adding)

For you example:
dest := 0
source := 1

**Comment by mateogianolio** [comment]
Closed by #138.
